app/routers/event.py

```diff
@@ -1,32 +1,33 @@
-from datetime import datetime as dt
 import json
+from datetime import datetime as dt
 from operator import attrgetter
 from typing import Any, Dict, List, Optional, Tuple
 import urllib
 
-from fastapi import APIRouter, Depends, HTTPException, Request
-from pydantic import BaseModel
-from sqlalchemy.exc import SQLAlchemyError
-from sqlalchemy.orm import Session
-from sqlalchemy.orm.exc import MultipleResultsFound, NoResultFound
-from sqlalchemy.sql.elements import Null
-from starlette import status
-from starlette.responses import RedirectResponse, Response
-from starlette.templating import _TemplateResponse
-
 from app.database.models import Comment, Event, User, UserEvent
 from app.dependencies import get_db, logger, templates
+from app.internal import comment as cmt
+from app.internal.emotion import get_emotion
 from app.internal.event import (
     get_invited_emails,
+    get_location_coordinates,
     get_messages,
     get_uninvited_regular_emails,
     raise_if_zoom_link_invalid,
 )
-from app.internal import comment as cmt
-from app.internal.emotion import get_emotion
 from app.internal.privacy import PrivacyKinds
 from app.internal.utils import create_model, get_current_user
 from app.routers.categories import get_user_categories
+from fastapi import APIRouter, Depends, HTTPException, Request
+from pydantic import BaseModel
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.orm import Session
+from sqlalchemy.orm.exc import MultipleResultsFound, NoResultFound
+from sqlalchemy.sql.elements import Null
+from starlette import status
+from starlette.responses import RedirectResponse, Response
+from starlette.templating import _TemplateResponse
```

Comment:
These lines should appear on the 2nd paragraph. Please use git precommit hooks to order it.

Reply:
Actually, precommit hooks didn't fix it automatically. I separated the paragraphs.